pkg/surveyext/multiselectwithadd.go

```diff
@@ -265,12 +264,6 @@ func (m *MultiSelectWithAdd) Prompt(config *survey.PromptConfig) (interface{}, e
 		}
 	}
 
-	// if there are no options to render
-	if len(m.Options) == 0 {
-		// we failed
-		return "", errors.New("please provide options to select from")
-	}
-
```

Comment:
this copy of the multi-select is specifically designed to allow adding new entries, an empty starting list should not be an error